### PR TITLE
[FEATURE2] sdfsdfdsfdsf

### PR DESCRIPTION
**Pull request by VladimirCW**
added new tests

### CODE REVIEW

File: feature1.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-sfdsfdsfds
+addfdsfdssfdsfdsfds
```
Comment (VladimirCW) [inline feature1.txt]: rename it

**Comment by DziubaAnastasiia** [comment]
add smth

File: feature1.txt (hunk 1 above)
Comment (VladimirCW) [reply feature1.txt]: done

Comment (VladimirCW) [reply feature1.txt]: OK

**Pull request merged**